internal/images/images.go

```diff
@@ -41,7 +42,9 @@ func GetLatestTag(
 		Strategy:   img.ParseUpdateStrategy(string(updateStrategy)),
 	}
 	if allowTags != "" {
-		vc.MatchFunc, vc.MatchArgs = img.ParseMatchfunc(allowTags)
+		vc.MatchFunc, vc.MatchArgs = img.ParseMatchfunc(
+			fmt.Sprintf("regexp:%s", allowTags),
```

Comment:
I didn't read the docs well enough...

https://argocd-image-updater.readthedocs.io/en/stable/configuration/images/#filtering-tags